pkg/analyzer/endpoints/blocks.go

```diff
@@ -0,0 +1,28 @@
+package endpoints
+
+import (
+	"github.com/np-guard/models/pkg/netset"
+)
+
+// a base struct to represent external endpoints, segments and rule block
+type ipBlock struct {
+	Block *netset.IPBlock
+	name  string
```

Comment:
what is the name of an IPBlock?

Reply:
please explain in comment what the name refers to or give example

Reply:
IPBlock is a base class:
for segments - its the segment name
for ruleBlock - its the ip string that the user put

Reply:
then maybe better to use explicit fields rather than base class?
with explicit field for segmentName on the segment type, and explicit field for ipBlockStr on the ipBlock type..

Reply:
fixed